docs/kratos/passwordless/06_code.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: one-time-code
+title: Passwordless authentication with One-Time Code
+sidebar_label: One-Time Code
```

Comment:
```suggestion
sidebar_label: One-time code
```

Regardless of `code` or `password` -> sentence casing or rather don't capitalize one-time code since its not an Ory specific term or product. Also in the rest of the doc.